src/screenshot-window.ts

```diff
@@ -12,16 +12,25 @@ export function createScreenshotWindow( captureUrl: string ) {
 		webPreferences: { session: newSession },
 	} );
 
-	const finishedLoading = new Promise< void >( ( resolve ) => {
-		window.webContents.on( 'did-finish-load', () => resolve() );
-	} );
```

Comment:
`window.loadURL` returns a promise that resolves when the `did-finish-load` event has fired, so I took the opportunity to simplify this logic.